### PR TITLE
fix: solve plots for the new version

### PR DESCRIPTION
**Pull request by MrtinoRG**

### CODE REVIEW

**Comment by kjappelbaum** [comment]
that is cool! We should see how we best can merge it with https://github.com/lamalab-org/chembench-paper/pull/38 where I am currently slowly rebuilding the workflow

**Comment by MrtinoRG** [comment]
No problem, I can move all the changes there and then close this PR

**Comment by kjappelbaum** [comment]
I think we can do different things and it is not clear which one is the best: 

- if you keep the changes here mostly in the scripts it is also very easy to merge yours into main and then update my PR 
- we can make this PR go into mine 
- I can try to be faster and merge into main first 
- we can only work on one branch (might be messy)

**Comment by MrtinoRG** [comment]
> I think we can do different things and it is not clear which one is the best:
> 
> * if you keep the changes here mostly in the scripts it is also very easy to merge yours into main and then update my PR
> * we can make this PR go into mine
> * I can try to be faster and merge into main first
> * we can only work on one branch (might be messy)

I have no preference, I can adapt to any of them

**Comment by AdrianM0** [comment]
I recommend dropping `wordcloud.py`

File: src/scripts/get_human_performance_dicts.py
Comment (kjappelbaum) [inline src/scripts/get_human_performance_dicts.py:1]: i think here we maybe both changed things

Comment (MrtinoRG) [reply src/scripts/get_human_performance_dicts.py:1]: I will revise this PR later, compare it with yours, and try to refactor stuff that is overlapping

**Comment by MrtinoRG** [comment]
I think we can close this one right @kjappelbaum ?

**Comment by kjappelbaum** [comment]
yeah - I think it is outdated now, unless you think that there is still something in there that we should keep

**Pull request closed without merging**